dev-requirements.txt

```diff
@@ -0,0 +1,257 @@
+#
+# This file is autogenerated by pip-compile with python 3.10
+# To update, run:
+#
+#    pip-compile dev-requirements.in
```

Comment:
Could you briefly update the README so it's clear that this is how requirements are installed now.

Reply:
💯 done!